Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -500,16 +500,16 @@ private bool SignBlock(List<StakeTx> stakeTxes, Block block, ChainedBlock pindex
 
                 for (uint n = 0; n < Math.Min(nSearchInterval, maxStakeSearchInterval) && !fKernelFound; n++)
                 {
+                    uint txTime = txNew.Time - n;
+                    if ((txTime & PosConsensusValidator.StakeTimestampMask) != 0)
```

Comment:
you forgot (~)
```
 if ((txTime & ~PosConsensusValidator.StakeTimestampMask) != 0)
```

Reply:
No :) 

The mask is there to only allow tiestamps divisible by 16. 
I'm here skipping such a timestamp that is not divisible by 16.
You would skip timestamps that are greater than 15.

Reply:
ok lets discuss on slack